src/apps/run/components/juttle-viewer.js

```diff
@@ -15,10 +15,14 @@ class JuttleViewer extends Component {
     }
 
     render() {
-        let juttleSource = false;
```

Comment:
I think changing for consistency sake was a good idea (the thing is called a bundle everywhere else). As for changing bundle to juttleSource, thats a debate for another place